src/HealthChecks.AzureServiceBus/AzureServiceBusHealthCheck.cs

```diff
@@ -3,50 +3,45 @@
 using Azure.Messaging.ServiceBus;
 using Azure.Messaging.ServiceBus.Administration;
 
-namespace HealthChecks.AzureServiceBus
+namespace HealthChecks.AzureServiceBus;
```

Comment:
Please revert switching to file-scoped namespaces in existing files to simplify review.

Reply:
Do you feel it's okay reverting it back to file-scoped namespaces before merge? Or should that kind of a change be a new PR? I don't know if it's already planned a bigger overhaul regarding stuff like this or what the guidelines are regarding this.

Reply:
Other PR. Such convertions not only make review harder but also inevitably break almost every pending PR that toutches that file. I would have converted all the files to use file-scoped namespaces if there were no PRs in this repository.